gtsam/navigation/PreintegratedRotation.h

```diff
@@ -21,12 +21,37 @@
 
 #pragma once
 
-#include <gtsam/geometry/Pose3.h>
 #include <gtsam/base/Matrix.h>
 #include <gtsam/base/std_optional_serialization.h>
+#include <gtsam/geometry/Pose3.h>
+#include "gtsam/dllexport.h"
 
 namespace gtsam {
 
+namespace internal {
+/**
+ * @brief Function object for incremental rotation.
+ * @param measuredOmega The measured angular velocity (as given by the sensor)
+ * @param deltaT The time interval over which the rotation is integrated.
+ * @param body_P_sensor Optional transform between body and IMU.
+ */
+struct GTSAM_EXPORT IncrementalRotation {
+  const Vector3& measuredOmega;
+  const double deltaT;
+  const std::optional<Pose3>& body_P_sensor;
```

Comment:
I am not sure I understand the point of making this a struct since the members here don't seem to be used beyond the `operator()` function call.
The biggest question I have is how would this affect inlining of the function? In terms of keyboard-typing, there is no benefit since we have to pass in the same number of variable (over 2 lines now instead of 1) so if this makes it hard for certain compilers to inline the function, then it may not be the best option.

Reply:
I opted to make it very clear that this is a function of the bias and all other arguments are parameters. I acknowledge it might affect inlining, although I have no idea how so I'm opting for clarity here.